slurm-advanced.tmpl

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
this message is coming from the one in `inst/`. I removed the top-level one and it still gives this error...